src/server/campaignd/server.cpp

```diff
@@ -449,45 +449,60 @@ void server::load_config()
 	}
 
 	LOG_CS << "Loaded addons metadata. " << addons_.size() << " addons found.\n";
+
+	load_tls_config(cfg_);
 }
 
 std::ostream& operator<<(std::ostream& o, const server::request& r)
 {
-	o << '[' << r.addr << ' ' << r.cmd << "] ";
+	o << (utils::get_if<tls_socket_ptr>(&r.sock) ? "TLS" : "") << '[' << r.addr << ' ' << r.cmd << "] ";
```

Comment:
Here I recommend `utils::holds_alternative` (alias for [`std::holds_alternative`](https://en.cppreference.com/w/cpp/utility/variant/holds_alternative)) since you're not using the value, only checking what type it holds.